ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKStreamTest.java

```diff
@@ -576,12 +656,83 @@ public void shouldPerformStreamToTableInnerJoin() {
                  joinedKStream.sourceSchemaKStreams);
   }
 
+  @Test
+  public void shouldSummarizeExecutionPlanCorrectly() {
+    // Given:
+    final SchemaKStream parentSchemaKStream = mock(SchemaKStream.class);
+    when(parentSchemaKStream.getExecutionPlan(anyString()))
+        .thenReturn("parent plan");
+    final SchemaKStream schemaKtream = new SchemaKStream(
+        simpleSchema,
+        mock(KStream.class),
+        simpleSchema.field("key"),
+        ImmutableList.of(parentSchemaKStream),
+        Serdes.String(),
+        Type.SOURCE,
+        ksqlConfig,
+        functionRegistry,
+        queryContext.push("source").getQueryContext());
+
+    // When/Then:
+    final String expected =
+        " > [ SOURCE ] | Schema: [key : VARCHAR, val : BIGINT] | Logger: query.node.source\n\t"
```

Comment:
nit: it may be easier to read if we use `equalToIgnoringWhiteSpace` so that newlines and tabs are ignored

Reply:
Actually we still want to make sure the tabs are put in correctly.